### PR TITLE
Create posts 2022-12-05 posts 

### PR DESCRIPTION
**Pull request by prycrane**

### CODE REVIEW

**Review by delisma — CHANGES_REQUESTED** [review]

File: _includes/footers/footer.html [hunk 1]
```diff
@@ -0,0 +1,134 @@
+<footer id="wb-info">
```
[the remark this answers is not in the document]
Comment (prycrane) [reply _includes/footers/footer.html]: I'm not sure how to do this.  You'll have to show me.

File: _layouts/post-01.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
+---
```
[the remark this answers is not in the document]
Comment (prycrane) [reply _layouts/post-01.html:1]: Same as above.

File: en/_posts/2022-11-29-wayfinding-project-improves-navigation.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
+---
+css:
+- https://design.canada.ca/css/split-h1.css
+- https://design.canada.ca/css/custom.css
+- https://use.fontawesome.com/releases/v5.15.4/css/all.css
```
[the remark this answers is not in the document]
Comment (prycrane) [reply en/_posts/2022-11-29-wayfinding-project-improves-navigation.md:5]: Not necessarily.  We can have this update for future blogs and keep the past blogs in its old look.  Greed that we should still do this at the template level.

File: en/_posts/2022-11-29-wayfinding-project-improves-navigation.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+layout: post-01
```
[the remark this answers is not in the document]
Comment (prycrane) [reply en/_posts/2022-11-29-wayfinding-project-improves-navigation.md:6]: We can discuss.  Not sure what you mean here.

File: en/_posts/2022-11-29-wayfinding-project-improves-navigation.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+altLangPage: /2022/12/05/
+date: 2022-11-29
+dateModified: 2022-11-29
```
[the remark this answers is not in the document]
Comment (prycrane) [reply en/_posts/2022-11-29-wayfinding-project-improves-navigation.md:9]: Not sure what you mean here.

File: en/_posts/2022-11-29-wayfinding-project-improves-navigation.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+description: "The wayfinding study investigated issues with Canada.ca navigation and informed changes to improve navigation and simplify the adoption of the Canada.ca design for departments and agencies.  The main improvements include the introduction of a new sign in button, removing the menu button, a contextual footer band, a main footer band with themes, and a simplified design for the theme page."
```
[the remark this answers is not in the document]
Comment (prycrane) [reply en/_posts/2022-11-29-wayfinding-project-improves-navigation.md]: Agreed.  I fixed this.

File: en/_posts/2022-11-29-wayfinding-project-improves-navigation.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+title: "The Wayfinding project improves Canada.ca navigation"
+---
+<h1 property="name" id="wb-cont" dir="ltr"><span class="stacked"><span>The Wayfinding project improves Canada.ca navigation</span>: <span>Canada.ca blog</span></span></h1>
+<div class="row">
+  <div class="col-md-8"> <img src="/images/thumbs/2022-11-29.png" class="img-responsive" alt="" /> </div>
+</div>
```
[the remark this answers is not in the document]
Comment (prycrane) [reply en/_posts/2022-11-29-wayfinding-project-improves-navigation.md:16]: Agree with the templating.  For new blogs only.  Perhaps?

File: en/_posts/2022-11-29-wayfinding-project-improves-navigation.md [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+<p>To find a solution we needed to look beyond the menu and toward navigation habits as a whole to see how institutions could meet user needs effectively.</p>
+<div class="row">
+  <div class="col-md-8 mrgn-tp-lg">
+    <blockquote class="mrgn-tp-lg">
+      <p>Navigation is about wayfinding, you can't treat it as separate because many other things run parallel with it. If you look at studies in wayfinding, everything from exhibit design to building the cathedrals, it's about creating a complete system. It's about looking at the whole.</p>
+      <p>– Clement Mok</p>
+    </blockquote>
+  </div>
+</div>
+<div class="clearfix"></div>
```
[the remark this answers is not in the document]
Comment (prycrane) [reply en/_posts/2022-11-29-wayfinding-project-improves-navigation.md:27]: Yeah.  Fixed.  No harm, no foul.  It's a bad habit I picked up.  :)

**Pull request closed without merging**